packages/ui-kit/src/VisitibilityType.ts

```diff
@@ -0,0 +1,3 @@
+export type VisibilityTypesInternal = 0;
```

Comment:
I know you tend to favor type unions instead of enums, but could you remind me of the reason? 😛